### PR TITLE
Grafana prometheus deployment tf

### PR DESCRIPTION
**Pull request by AndyBoWu**
added steps to start grafana service at boot

### CODE REVIEW

File: grafana-deployment/main.tf [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    Name    = "Grafana-tf"
+    Project = "Harmony"
+  }
+
+  volume_tags = {
+    Name    = "HarmonyNode-monitoring-Volume"
+    Project = "Harmony"
+  }
+
+  # provisioner "file" {
```
Comment (LeoHChen) [inline grafana-deployment/main.tf]: remove commented lines.

File: grafana-deployment/terraform.tfstate [hunk 2]
```diff
@@ -0,0 +1,187 @@
+{
```
Comment (LeoHChen) [inline grafana-deployment/terraform.tfstate]: don't check-in the tfstate.

File: grafana-deployment/terraform.tfstate.backup [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (LeoHChen) [inline grafana-deployment/terraform.tfstate.backup]: don't check in the backup file.

File: grafana-deployment/variables.tf [hunk 4]
```diff
@@ -0,0 +1,45 @@
+# variables for ec2 instances
+
+variable "private_key_path" {
+  description = "The path to the SSH Private Key to access ec2 instance."
+  default     = "/Users/bwu2/Harmony/keys/california-key-benchmark.pem"
```
Comment (LeoHChen) [inline grafana-deployment/variables.tf]: don't add your home path, you may use ~/.ssh/keys

File: grafana-deployment/variables.tf [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+variable "node_volume_size" {
+  description = "Root Volume size of the ec2 node instance"
+  default     = 250
+}
+
+variable "node_instance_type" {
+  description = "Instance type of the ec2 node instance"
+  default     = "m5d.large"
```
Comment (LeoHChen) [inline grafana-deployment/variables.tf]: don't use m5d.large, use m5a.large

**Comment by AndyBoWu** [comment]
we will have a dockerized solution for this project.

**Pull request closed without merging**